### PR TITLE
Trim .git from project in taskgraph.parameters._get_defaults

### PR DESCRIPTION
**Pull request by gabrielBusta**
The untrimmed project name from this function is breaking `taskgraph test-action-callback`

### CODE REVIEW

**Review by jcristau — APPROVED** [review]

File: src/taskgraph/parameters.py [hunk 1]
```diff
@@ -80,6 +80,8 @@ def _get_defaults(repo_root=None):
     try:
         repo_url = repo.get_url()
         project = repo_url.rsplit("/", 1)[1]
+        if project.endswith(".git"):
+            project = project[:-4]
```
Comment (JohanLorenzo) [inline src/taskgraph/parameters.py:84]: I'm glad this problem comes at the same time as `mozilla-repo-urls` got out. https://github.com/taskcluster/taskgraph/pull/107 introduced it. Let's leverage it here. Once that PR is done, we can bump `mozilla-repo-urls` to 0.0.3 and use this piece of code:

```suggestion
        repo_url = repo.get_url()
        parsed_url = parse(repo_url)
        project = parsed_url.repo_name
```

What do you think, @gabrielBusta?

Comment (gabrielBusta) [reply src/taskgraph/parameters.py:84]: I think it's great :) I can bump it in this PR and get rid of this repetitive logic while Im at it

Comment (JohanLorenzo) [reply src/taskgraph/parameters.py:84]: Awesome! mozilla-repo-urls 0.0.3 has been published! 🚢 

https://pypi.org/project/mozilla-repo-urls/0.0.3/#files

**Review by ahal — CHANGES_REQUESTED** [review]
Sounds like we're planning to use `mozilla-repo-urls` here, so requsting changes for that.

**Review by ahal — APPROVED** [review]

**Comment by JohanLorenzo** [comment]
🚀 \o/

**Pull request merged**